packages/autorag/frontend/src/app/components/run-results/PatternDetailsModal/PatternDetailsModal.scss

```
@@ -0,0 +1,125 @@
+.autorag-pattern-details-modal {
+  --pf-v6-c-modal-box--MaxWidth: 1300px;
+  --pf-v6-c-modal-box--m-lg--lg--MaxWidth: 1300px;
+  height: 80vh;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)